src/main/java/org/folio/ebsconet/mapper/OrdersMapper.java

```diff
@@ -95,9 +94,11 @@ public void ebsconetToFolio(MappingDataHolder mappingDataHolder) {
     populateCostAndLocations(poLine, ebsconetOrderLine);
     removeZeroAmountLocations(poLine);
 
-    if (ebsconetOrderLine.getType() == NON_RENEWAL)
-      cancelOrderLine(poLine);
-
+    if (ebsconetOrderLine.getType()!= null) {
```

Comment:
please combine two `if` blocks into single one for better readability

Reply:
sure